### DISCUSSION TITLE
We are merging PRs without a Review

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-openapi#1295] [General]
We merged the following PRs: 

- https://github.com/micronaut-projects/micronaut-openapi/pull/1293
- https://github.com/micronaut-projects/micronaut-openapi/pull/1289/files
- https://github.com/micronaut-projects/micronaut-openapi/pull/1282/files
- https://github.com/micronaut-projects/micronaut-openapi/pull/1281/files

without a Review. 

we should accept PRs which change production code without a review from someone. 

We do exceptions. for example: If you write a PR which improves a test or updates a dependency version, it can be merged without a review. But a PR changing production code should be reviewed by someone.

**Comment by sdelamo** [comment]
@altro3 Please note that we appreciate your contributions a lot. 

Could you please ask me as a reviewer in your PRs? I will try to be on top of them and do not make you wait.

**Comment by altro3** [comment]
@sdelamo Yes, I understand that the last merges were not according to the protocol. But there were fairly minor fixes, I would say, routine ones. It seemed to me that I could do this without review, because... As you correctly noted, the review can take several weeks.

Okay, I'll add you to merge requests.

**Reply by sdelamo** [reply]
Thanks for your patience.
